src/fsharp/InfoReader.fs

```diff
@@ -783,4 +814,67 @@ let PropTypOfEventInfo (infoReader: InfoReader) m ad (einfo: EventInfo) =
     let argsTy = ArgsTypOfEventInfo infoReader m ad einfo 
     mkIEventType g delTy argsTy
 
+let private RuntimeSupportsFeature (infoReader: InfoReader) m featureId =
+    let g = infoReader.g
+
+    let runtimeFeature =
+        match featureId with
+        | LanguageFeature.DefaultInterfaceMethodConsumption -> "DefaultImplementationsOfInterfaces"
+        | _ -> String.Empty
+
+    if String.IsNullOrWhiteSpace runtimeFeature then
+        true
+    else
+        match g.System_Runtime_CompilerServices_RuntimeFeature_ty with
+        | Some runtimeFeatureTy ->
+            infoReader.GetILFieldInfosOfType (Some runtimeFeature, AccessorDomain.AccessibleFromEverywhere, m, runtimeFeatureTy)
+            |> List.exists (fun ilFieldInfo -> ilFieldInfo.FieldName = runtimeFeature)
+        | _ ->
+            false
+
+[<Struct>]
```

Comment:
I doubt this needs to be a struct, generally only put Struct if you're confident it's needed?

Reply:
Does not need to be, I imagine we could be doing a lot of calls to getting LanguageFeatureSupport overall and it didn't hurt to make it a Struct. Two reference type fields, on a 64-bit machine, 16 bytes is the upper limit for a struct.

But, I'll switch it back to a reference type because in reality it won't be an issue.